src/blockchain_state_channel_v1.proto

```diff
@@ -31,7 +31,7 @@ message blockchain_state_channel_v1 {
 
 message blockchain_state_channel_response_v1 {
     bool accepted = 1;
-    packet downlink = 2;
```

Comment:
Just for my knowledge, the omission of other old fields (in numbering) is fine I'm assuming? This is just so that we don't break previously configured messges. Right?

Reply:
yes